### ISSUE TITLE
Cannot reuse statements

### ISSUE THREAD

**Issue by ueffel** [sijms/go-ora#337]
Same test code as in #172, but different error this time.

DBs:
* `Oracle Database 12c Enterprise Edition Release 12.1.0.2.0 - 64bit Production`
* `Oracle Database 19c Enterprise Edition Release 19.0.0.0.0 - Production
Version 19.17.0.0.0`

affected versions: v2.6.9, v2.6.10

v2.6.8 is working fine.

test code:

```go
package main

import (
	"database/sql"
	"log"

	_ "github.com/sijms/go-ora/v2"
)

func must(err error) {
	if err != nil {
		panic(err)
	}
}

func main() {
	db, err := sql.Open("oracle", "oracle://USER:PW@ADDR/SERVICE")
	must(err)
	defer db.Close()
	stmt, err := db.Prepare("select :s from dual")
	must(err)
	defer stmt.Close()

	rows, err := stmt.Query("hi")
	must(err)

	var s string
	for rows.Next() {
		err = rows.Scan(&s)
		must(err)
		log.Println(s)
	}
	must(rows.Err())
	rows.Close()

	rows2, err := stmt.Query("hello")
	must(err) // panic: ORA-03120: two-task conversion routine: integer overflow
	for rows2.Next() {
		err = rows2.Scan(&s)
		must(err)
		log.Println(s)
	}
	must(rows2.Err())
	rows2.Close()
}
```

results in a panic:

```plain
panic: ORA-03120: two-task conversion routine: integer overflow
```

**Comment by sijms** [comment]
fixed in v2.6.11

**Comment by ueffel** [comment]
Seems to be fixed 🎉
